src/components/MapComponent.vue

```diff
@@ -130,6 +145,40 @@ export default {
         closeOnClick: true
       })
     },
+    createMultipleSelectPopup() {
+      this.selectionPopup = new mapboxgl.Popup({
+        closeButton: true,
+        closeOnClick: true
+      })
+    },
+
+    writePopUp(elem) {
```

Comment:
not very descriptive. Please add comments (try to do that for each new function at least and where code needs explaining)